Entity/Notification.php

```diff
@@ -45,11 +45,14 @@
      */
     protected $createdAt;
 
-    public function __construct()
+    public function __construct(string $title, string $message, array $recipients)
     {
         $this->id              = Uuid::uuid4();
         $this->recipientsItems = new ArrayCollection();
         $this->createdAt       = new \DateTime();
+        $this->setTitle($title);
+        $this->setMessage($message);
+        array_walk($recipients, function ($recipient) {$this->addRecipient($recipient);});
```

Comment:
something with codestyle.
Btw, does `this` work in the scope of lambda function? I am not sure about it -- https://3v4l.org/BK2dp

Reply:
![image](https://user-images.githubusercontent.com/2415637/32837526-8db55c78-ca16-11e7-8a74-b39b1a73d4af.png)

Reply:
works